tests/acceptance/features/apiSearch/apiSpaceSearch.feature

```diff
@@ -131,8 +131,8 @@ Feature: Search
     When user "Brian" searches for "folder" inside folder "/folderMain" in space "Shares" using the WebDAV API
     Then the HTTP status code should be "207"
     And the search result of user "Brian" should contain only these entries:
-      | /SubFolder1                                |
-      | /SubFolder1/subFOLDER2                     |
-      | /SubFolder1/subFOLDER2/insideTheFolder.txt |
+      | folderMain/SubFolder1                                |
+      | folderMain/SubFolder1/subFOLDER2                     |
+      | folderMain/SubFolder1/subFOLDER2/insideTheFolder.txt |
```

Comment:
missing changes in other scenarios:
`Scenario Outline: search files inside the folder`
`Scenario Outline: sharee cannot find resources that are not shared`